assets/scene/lang.lua

```diff
@@ -3,13 +3,19 @@ local langList={
     zh_full="全简体中文",
     zh_trad="繁體中文",
     en="English",
+    en_googled="English Language",
```

Comment:
You don't need to add unused languages, to be honest
I thought you would sync the language list to keep up with Techmino, but you added language slots that I don't even know if someone have enough motivation to do ;-;